### PR TITLE
iOS export: invalid identifier, the character '-' is not allowed

### PR DESCRIPTION
**Pull request by njt1982**
I tried to set my identifier to co.uk.thingy-ma-jig.MyGameName and I get this:

image

Except... hyphens are valid. The Apple portal didn't complain when I setup the App ID bundle with hypens.

### CODE REVIEW

**Comment by njt1982** [comment]
@KoBeWi how is this?

**Comment by akien-mga** [comment]
The change looks good, but the commit log should be amended (with `git commit --amend`, then `git push --force origin 30050-identifier-fix` to update the PR) to be made more explicit.

Commit logs should explain in plain English what the functional change is, e.g. here "Allow '-' character in iOS reverse DNS identifiers".

**Comment by njt1982** [comment]
@akien-mga thanks for the ammend/force-push tip (never thought about doing that, but obviously it's possible!).

This is done. Great to see such care and attention on PR review!

**Review by akien-mga — APPROVED** [review]

**Review by volzhs — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks and congrats for your first Godot PR :)
